### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by szepeviktor**
### Description of the Change

Found few misspellings.

### CODE REVIEW

File: inc/classes/class-srm-redirect.php [hunk 1]
```diff
@@ -241,8 +241,8 @@ public function match_redirect( $requested_path ) {
 				 * @param {string} $redirect_url Final URL to redirect to.
 				 * @returns {string} Final URL to redirect to.
 				 */
-				$filterd_redirect_to   = apply_filters( 'srm_redirect_to', $redirect_to );
-				$sanitized_redirect_to = esc_url_raw( $filterd_redirect_to );
+				$filtered_redirect_to  = apply_filters( 'srm_redirect_to', $redirect_to );
```
Comment (jeffpaul) [inline inc/classes/class-srm-redirect.php:244]: I'm a little worried making this change in case someone's already using the misspelled filter

Comment (szepeviktor) [reply inc/classes/class-srm-redirect.php:244]: The filter name is unchanged: `srm_redirect_to` 💡

**Review by iamdharmesh — APPROVED** [review]
Thanks for fixing typos @szepeviktor. LGTM.

**Comment by szepeviktor** [comment]
Glad to contribute.

**Pull request merged**